flink-streaming-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/config/AWSConfigConstants.java

```diff
@@ -40,7 +40,10 @@
 		PROFILE,
 
 		/** Simply create AWS credentials by supplying the AWS access key ID and AWS secret key in the configuration properties */
-		BASIC
+		BASIC,
+
+		/** Don't supply AWS credentials and rely on the AWS library auto-detecting, which supports ENV vars and AWS MetaData **/
```

Comment:
"which supports ENV vars and AWS MetaData" -> I think the auto-detecting checks more than env vars.

Can we change this to something like, "A credentials provider chain will be used that searches for credentials in this order: ENV_VARS, SYS_PROPS, PROFILE in the AWS instance metadata". According to the AWS API Javadocs, this is the actual behaviour.